### ISSUE TITLE
Problem with mapping of Preferred ID

### ISSUE THREAD

**Issue by kateblanch** [american-art/aac_mappings#46]
I expected to see:

34753

but instead I saw:

Nothing/Blank

The current query was:

```ttl
PREFIX owl: <http://www.w3.org/2002/07/owl#>
PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX crm: <http://www.cidoc-crm.org/cidoc-crm/>
PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
PREFIX foaf: <http://xmlns.com/foaf/0.1/>
PREFIX dct: <http://purl.org/dc/terms/>
PREFIX dc: <http://purl.org/dc/elements/1.1/>
PREFIX schema: <http://schema.org/>

SELECT ?identifier ?identifier_url
WHERE {
<http://data.thewalters.org/object/34753> a crm:E22_Man-Made_Object .
<http://data.thewalters.org/object/34753> crm:P1_is_identified_by ?identifier_url .
?identifier_url a crm:E42_Identifier ;
  rdf:value ?identifier ;
  crm:P2_has_type <http://vocab.getty.edu/aat/300404670> .

}


```

**Comment by workergnome** [comment]
It does not appear that any IDs were mapped—they don't appear in the data.

**Comment by kateblanch** [comment]
I don't see a mapping to Object ID in the Object Model. I'll open an issue under the WAM repo.
If there's a better way to address this, let me know :)

**Comment by workergnome** [comment]
Nope--that's the right thing to do!

**Comment by bsnikhila** [comment]
I need a little help understanding which column I should model as what.
Wam (and most other museums I think) have 2 columns - Object ID and Object Number.
The review site has models for 3 fields - Preferred ID, Object ID and Accession Number.
Which should be modeled as what?

**Comment by workergnome** [comment]
Accession number should be an explicit field—it's a museum term, and it's typically a human-readable ID based on the year and order the painting was acquired in.   It's _possible_, though I'm not sure, that Object Number means accession number for TMS-based institutions.

Really, this is going to be up to the institution, and may vary between them.

**Comment by ddegler** [comment]
I think this was discussed at length and resolved in the following issue thread: https://github.com/american-art/aac_mappings/issues/4 

Not sure if the Accession Number was the final decision, or whether there are institutional differences. Both an Object ID and Accession Number are useful identifiers for the institutions, so the data items should not be lost. One will also be considered Preferred, but as David says this may be specified by each institution.

**Comment by kateblanch** [comment]
I will open an issue under our WAM repo - the bigger problem is no ids (preferred or object) are showing up in our data, but that might be WAM specific. 

Thanks for the reminder of the other issue, I remember this now!
Under issue #4  I did post the responses I got from partners about which data point would ideally be assigned to which id (preferred id or object id)...not sure anything ever came of it, but at least it's documented.
